edb/edgeql/compiler/func.py

```diff
@@ -245,6 +245,11 @@ def compile_FunctionCall(
     else:
         tuple_path_ids = []
 
+    impl_is_strict = (
+        func.get_impl_is_strict(env.schema)
+        and func.get_language(env.schema) != qlast.Language.EdgeQL
```

Comment:
Can you add a comment explaining the `and != edgeql` part?

Reply:
Reworked this bit away